rel-eng/tito.props

```diff
@@ -407,6 +408,7 @@ whitelist = puppet-foreman_scap_client
   rubygem-algebrick
   rubygem-apipie-params
   rubygem-chef-api
+  rubygem-concurrent-ruby
```

Comment:
Already in the `foreman-nightly-nonscl-rhel7` list, it doesn't need to be built for `foreman-plugins-nightly-nonscl-rhel7` too.